src/alias/apache

```diff
@@ -0,0 +1 @@
+parse "* - * [*] \"* * *\" * *" as ip, name, ts, method, url, protocol, status, cl
```

Comment:
can you do `contentlength` for `cl`? I think it's a bit too esoteric to be an acronym

Reply:
Done!